docs/use_cases/retrieval_augmented_generation_eval.md

```diff
@@ -0,0 +1,180 @@
+RAG becomes crucial when users seek to generate new content rather than interact with documents or search results directly. 
+It excels in providing contextually rich, informative, and human-like responses, making it an ideal solution for a comprehensive and versatile approach when integrated with vector search.
+
+Your next step is to conduct feasibility studies and correlate the results with your business needs and value expectations.
+
+## Evaluating Retrieval-Augmented Generation
+
+Our experience scaling RAG from Proof-of-concept (POC) to production taught us
+about the unknowns. The lesson is to improve and optimize your solution through evaluation.
+
+### Why do you need to evaluate?
+
+Evaluation is a very crucial step to distill a sense of Trust in your application, which establishes reputation, and boosts team morale and confidence.
+Needless to say, it also validates that your applications avoid common pitfalls, and it is no different Evaluation is key. It distills a sense of Trust in your application,
+which establishes reputation, and boosts team morale and confidence.
+It also validates that your applications avoid common pitfalls.
+The aspect of relevance assessment through DCG / nDCG or machine learning model through train-validation-test split/overfitting/underfitting analysis addressed similar concerns.
+
+And for some who think LLMs are perfect, they can and do make mistakes too!! 🙂
+One problem is that the 'data' frequently changes. Evaluation helps ensure the results are consistent with user expectations.
+
+### If you can't quantify it, you can't improve it!!
+
+<br>For RAG it can be restated as :
+
+### If you can't retrieve it, you can't generate it!!
+
+Our experience has taught us that it is better to evaluate each component in isolation.
+
+As for different components of RAG viz. Information Retrieval - Context Augmentation - Response Generation , we classified challenges of RAG as below :
+
+<img src=/assets/use_cases/retrieval_augmented_generation_eval/rag_challenges.png alt="Classification of Challenges of RAG Evaluation" data-size="100" />
+
+For evaluation, we need metrics/methodology/frameworks that uses one or more
+tools. The tools that you use should cover each component of RAG. Such coverage
+ensures granular and thorough measurements.
+
+Evaluation metrics can assess:
+
+- Retrieval effectiveness
+- Coherence of generated responses
+- Relevance to the retrieved information.
+
+<img src=/assets/use_cases/retrieval_augmented_generation_eval/rag_granular.jpg alt="Granular Levels of Evaluation of RAG" data-size="100" />
+
+### Let's dive in !!
+
+## Strategies for Evaluation
+
+Now that we have defined different challenges and levels at which we can break down the RAG evaluation, let us zoom into the levels individually. 
+
+### Model Evaluation 
+
+To evaluate our model, we start with this [Massive Text Embedding Benchmark](https://github.com/embeddings-benchmark/mteb#leaderboard). We want to ensure that
```

Comment:
How about putting a direct link to the leaderboard?
https://huggingface.co/spaces/mteb/leaderboard